src/main/scala/com/thenewmotion/chargenetwork/ocpp/charger/BosService.scala

```diff
@@ -109,7 +129,34 @@ class ConnectorServiceImpl(protected val service: SyncCentralSystem, connectorId
     service(MeterValuesReq(ConnectorScope(connectorId), Some(transactionId), List(meter)))
   }
 
+  def signedAblMeterValue(transactionId: Int, meterValue: Int): Unit = {
+    service(MeterValuesReq(ConnectorScope(connectorId), Some(transactionId), List(signedMeterValuesBeginTransaction(meterValue))))
+  }
+
+  def signedMeterValuesBeginTransaction(meter: Int): Meter = {
+    val meterValue: Value = Value(value = meter.toString,
+      context = ReadingContext.TransactionBegin,
+      format = ValueFormat.Raw,
+      measurand = Measurand.EnergyActiveImportRegister,
+      phase = None,
+      location = Location.Outlet,
+      unit = UnitOfMeasure.Kwh)
+
+    val signedMeterReading: Value = meterValue.copy(context = ReadingContext.TransactionBegin,
+      format = ValueFormat.SignedData, value = signedMeter)
+
+    Meter(ChargerClock.now, List(meterValue, signedMeterReading))
+  }
+
+  def signedMeter: String = <values>
+    <value>
+      <signedData
+      format="ALFEN">AP;0;3;ALCV3ABBBISHMA2RYEGAZE3HV5YQBQRQAEHAR2MN;BIHEIWSHAAA2W2V7OYYDCNQAAAFACRC2I4ADGAETI4AAAABAOOJYUAGMXEGV4AIAAEEAB7Y6AAO3EAIAAAAAAABQGQ2UINJZGZATGMJTGQ4DAAAAAAAAAACXAAAABKYAAAAA====;R7KGQ3CEYTZI6AWKPOA42MXJTGBW27EUE2E6X7J77J5WMQXPSOM3E27NMVM2D77DPTMO3YACIPTRI===;
```

Comment:
For 2.0 Thomas is sending only the value (so from AP onwards) instead of the whole xml thing. Not sure if it is the correct but at least let's be consistent

Reply:
So should I send only `AP;0;3;ALCV3ABBBISHMA2RYEGAZE3HV5YQBQRQAEHAR2MN;BIHEIWSHAAA2W2V7OYYDCNQAAAFACRC2I4ADGAETI4AAAABAOOJYUAGMXEGV4AIAAEEAB7Y6AAO3EAIAAAAAAABQGQ2UINJZGZATGMJTGQ4DAAAAAAAAAACXAAAABKYAAAAA====;R7KGQ3CEYTZI6AWKPOA42MXJTGBW27EUE2E6X7J77J5WMQXPSOM3E27NMVM2D77DPTMO3YACIPTRI===` to ocpp? 

@fronto can you confirm that as well? I just want to make sure keeping consistency between the 2 versions won't cause any side effect on ocpp 1.6 simulation

Reply:
It seemed like common sense to me. The signature is the part inside the signedData element. Originally Alexander Mordashov suggested we just use some random bytes...that is what lead me to taking that approach